outside.go

```diff
@@ -455,9 +455,6 @@ func (f *Interface) handleRecvError(addr *udp.Addr, h *header.H) {
 		return
 	}
 
-	hostinfo.Lock()
-	defer hostinfo.Unlock()
```

Comment:
Why don't we need this anymore? Shouldn't be at least doublecheck that someone else didn't already handle a recv_error for this hostinfo on a different thread?